holidays/groups/hindu.py

```diff
@@ -306,6 +318,36 @@ def _add_thaipusam(self, name) -> Optional[date]:
             name, self._hindu_calendar.thaipusam_date(self._year)
         )
 
+    def _add_thiruvalluvar_day(self, name) -> Optional[date]:
+        """
+        Add Thiruvalluvar Day and Mattu Pongal.
+
+        Thiruvalluvar Day and Mattu Pongal are celebrated in Tamil Nadu, India, as part
+        of the Pongal festival. Thiruvalluvar Day honors the classical Tamil poet and
+        philosopher Thiruvalluvar, while Mattu Pongal is dedicated to cattle, recognizing
+        their importance in agriculture. Both events usually fall on January 15th or 16th
+        each year during the Tamil month of Thai.
+        https://en.wikipedia.org/wiki/Thiruvalluvar_Day
+        https://en.wikipedia.org/wiki/Pongal_(festival)#Mattu_Pongal
+        """
+        return self._add_hindu_calendar_holiday(
+            name, self._hindu_calendar.thiruvalluvar_day_date(self._year)
+        )
+
+    def _add_uzhavar_thirunal(self, name) -> Optional[date]:
+        """
+        Add Uzhavar Thirunal.
+
+        Uzhavar Thirunal is a harvest festival celebrated in Tamil Nadu, India,
+        as part of the Pongal festivities. It is dedicated to honoring farmers
+        (uzhavar) and their contribution to agriculture. Uzhavar Thirunal usually
+        falls on January 16th or 17th each year.
+        https://en.wikipedia.org/wiki/Pongal_(festival)#Uzhavar_Thirunal
+        """
+        return self._add_hindu_calendar_holiday(
+            name, self._hindu_calendar.uzhavar_thirunal_date(self._year)
```

Comment:
```suggestion
            name, self._hindu_calendar.pongal_date(self._year), days_delta=+2
```

Reply:
And update `_add_hindu_calendar_holiday()` method as
```python
    def _add_hindu_calendar_holiday(
        self, name: str, dt_estimated: tuple[Optional[date], bool], days_delta: int = 0
    ) -> Optional[date]:
        """
        Add Hindu calendar holiday.

        Adds customizable estimation label to holiday name if holiday date
        is an estimation.
        """

        return self._add_eastern_calendar_holiday(
            name, dt_estimated, self._hindu_calendar_show_estimated, days_delta
        )
```
Then we don't need THIRUVALLUVAR_DAY and UZHAVAR_THIRUNAL in calendar.